packages/minimongo/local_collection.js

```diff
@@ -1910,10 +1910,9 @@ const NO_CREATE_MODIFIERS = {
 };
 
 // Make sure field names do not contain Mongo restricted
-// characters ('.', '$', '\0').
+// characters ('.', '\0').
 // https://docs.mongodb.com/manual/reference/limits/#Restrictions-on-Field-Names
```

Comment:
This PR looks good to me. However, I'm curious about the explicit prohibition of the dollar sign in minimongo. The documentation attached to the code mentions field name restrictions in Mongo, so it might just be an implementation based on those rules on the past, without other reasons behind.

Reply:
Thank you for the context provided. Now, I remember some use cases faced on the past with "$" and ".". It is ok to have these unlocked.